### ISSUE TITLE
Re-write the base logic using StackOverflow.jl

### ISSUE THREAD

**Issue by logankilpatrick** [JuliaLangSlack/StackOverflowBot#18]
See StackOverflow.jl and use it to re-write this in Julia.

**Comment by logankilpatrick** [comment]
This is done and once Slack.jl is merged into the registry I need to make this repo work with Julia and not Python.
